ai_models/migrations/0003_videomodelspec_category.py

```
@@ -0,0 +1,22 @@
+
+    operations = [
+        migrations.AddField(
+            model_name="videomodelspec",
+            name="category",
+            field=models.IntegerField(
+                choices=[(1, "Video"), (2, "Audio")],
+                help_text="Model category: generates Audio, Video, etc.",
+                null=True,
+            ),
+        ),
+    ]
```

Ruff (0.14.1)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)